src/config/2022.json

```diff
@@ -577,6 +577,16 @@
       "avatar_url": "5749508",
       "github": "GJFR"
     },
+    "webmatter-it": {
+      "name": "Giulia Laco",
+      "teams": [
+        "translators"
+      ],
+      "avatar_url": "6397771",
+      "website": "https://www.webmatter.it/",
+      "github": "webmatter-it",
+      "twitter": "webmatter_it"
+    },
```

Comment:
@webmatter-it-it could you confirm you're all happy with this?

Here's what it looks like: https://20221025t125922-dot-webalmanac.uk.r.appspot.com/it/2022/fonts

And here's what you'll look like on our contributors page: https://20221025t125922-dot-webalmanac.uk.r.appspot.com/it/2022/contributors#webmatter-it

Reply:
Yes, great, thank you. If you fancy you can add my "linkedin": "giulialaco"